providers.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "=> 4.0"
+    }
+  }
+}
+
+provider "aws" {
```

Comment:
Delete this block. Modules should not have provider blocks.

Reply:
Deleted